src/System.Transactions/src/System/Transactions/CommittableTransaction.cs

```diff
@@ -42,6 +44,12 @@ internal CommittableTransaction(IsolationLevel isoLevel, TimeSpan timeout) : bas
             }
         }
 
+        // This serializtion constructor needs to be removed when we implement distributed transaction capabilities
```

Comment:
Thanks for the explanation.

> I can't make these "temporary" constructors protected, as suggested by serialization documentation, because the containing classes are sealed, except for Transaction.

I know we agreed on this approach of adding these constructors in the other issue thread.  But given your explanation here (and maybe I just didn't pay enough attention previously), it seems like the best bet may just be to suppress the FxCop warning, rather than adding these constructors.  The ctors aren't functional anyway, because attempting to serialize will throw, and they're private and unused.  And if we make Transaction's ctor protected, that's essentially adding new exposed API, and if we make it internal, we're probably still going to get an FxCop warning about it.